drivers/mcp2515/mcp2515.c

```diff
@@ -0,0 +1,412 @@
+/*
+ * Copyright (C) 2016 OTA keys
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     drivers_mcp2515
+ * @{
+ *
+ * @file
+ * @brief       mcp2515 can spi driver
+ *
+ * @author      Toon Stegen <toon.stegen@altran.com>
+ * @}
+ */
+
+#include <string.h>
+
+#include "xtimer.h"
+#include "mcp2515.h"
+#include "mcp2515_spi.h"
+#include "mcp2515_defines.h"
+
+#include "can/candev.h"
+
+#define ENABLE_DEBUG (0)
+#include "debug.h"
+
+/* reset delay should be at least 2 microseconds */
+#define RESET_DELAY_US 2
+
+/* size of transmission and reception buffers in mcp2515 */
+#define BUFFER_SIZE 13
+
+/* macros for getting the TX and RX control registers */
+#define TX_CTRL(mailbox) ((MCP2515_TXB0CTRL) + ((mailbox) << 4))
+//#define RX_CTRL(mailbox) ((MCP2515_RXB0CTRL) + ((mailbox) << 4))
```

Comment:
Why is this commented out?

Reply:
I commented this out recently, because Murdock signaled it as a redefinition of a define. (apparently the same name is used in some boardfiles)

I noticed that only RX_CTRL was a redefine, and I also noticed that it currently was not used in the drivercode. At first I thought about simply removing the line, but I ended up only commenting it out in case someone wanted to change this driver in the future and can still use this macro.

Reply:
Then may I suggest renaiming both to: `MCP2515_TX_CTRL()` and `MCP2515_RX_CTRL()`, would that solve it?